parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -327,8 +328,22 @@ notifyUser dir o = case o of
           <> "or invalid codebase, or because files inside the codebase"
           <> "are being deleted external to UCM."
     ]
-  MetadataAmbiguous _ppe [] -> pure . P.warnCallout .
+
+  LinkDefnsNotFound [path] -> pure . P.warnCallout .
+    P.wrap $ "I could not link the definition " <> (P.backticked . P.string . show) path <>
+    " as I could not find it in the codebase."
+
+  LinkDefnsNotFound paths -> pure . P.warnCallout .
+    P.wrap $ "I could not link the definitions " <> commaSeparated <> " as I could not find them in the codebase."
+    where
+    commaSeparated = P.commas (P.backticked . P.string . show <$> paths)
```

Comment:
@aryairani has already clocked that I am adding one too many spaces here `('a'<space><comma>)`, I am not 100% sure why, will aim to sort this out so in the meantime if someone happens to notice what I am doing wrong, by all means let me know.